documentation/modules/deploying/proc-deploy-access-operator.adoc

```diff
@@ -0,0 +1,43 @@
+// This assembly is included in the following assemblies:
+//
+// security/assembly-access-operator.adoc
+
+[id='proc-deploy-access-operator-{context}']
+= Deploying the Access Operator (early access)
+
+[role="_abstract"]
+This procedure shows how to deploy the Access Operator for access management of a Kafka cluster.
+You cannot use the Access Operator with a Kafka cluster that is not managed by the Cluster Operator.  
+
+.Prerequisites
+
+* Installation requires a Kubernetes user with `cluster-admin` role, such as `system:admin`.
+* The Access Operator installation files must be downloaded and extracted from Strimzi's link:https://github.com/strimzi/kafka-access-operator[`kafka-access-operator` GitHub repository^]. The installation files are contained in the `./install` directory.
+* xref:deploying-cluster-operator-str[The Cluster Operator must be deployed.]
+
+By default, the Access Operator is deployed in the `strimzi-access-operator` namespace. 
```

Comment:
Maybe not a question for you but for @katheris  ... why the installation files include a `Namespace` custom resource (which is where the operator is going to be installed) instead of letting the user have the namespace ready and specify it when applying the YAML? I mean it's not something we have for the cluster operator installation  so I was curious about this decision.

Reply:
Because it is cluster-wide out of the box. Cluster Operator is namespace scoped by default.